### PR TITLE
Event brite order api

### PR DESCRIPTION
**Pull request by rfeseha**
I changed the namespaces and also made some changes in one or two places. You guys can review my code and let me know if I should merger it

### CODE REVIEW

File: OrderApi/Models/OrderItem.cs [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+namespace EventBriteAssignment3A.Services.OrderApi.Models
+{
+    public class OrderItem
+    {
+        [Key]
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public int Id { get; set; }
+         
+         public string ProductName { get; set; }
```
Comment (vlau26) [inline OrderApi/Models/OrderItem.cs]:
```suggestion
         public string EventName { get; set; }
```

File: OrderApi/Models/OrderItem.cs [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        public string PictureUrl { get; set; }
+        public decimal UnitPrice { get; set; }
+        
+        public int Units { get; set; }
+        public int ProductId { get; private set; }
```
Comment (vlau26) [inline OrderApi/Models/OrderItem.cs]:
```suggestion
        public int EventId { get; private set; }
```

File: OrderApi/Models/OrderItem.cs [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+        protected OrderItem() { }
+        public Order Order { get; set; }
+        public int OrderId { get; set; }
+
+        public OrderItem(int productId, string productName, decimal unitPrice,   string pictureUrl, int units = 1)
```
Comment (vlau26) [inline OrderApi/Models/OrderItem.cs]:
```suggestion
        public OrderItem(int eventId, string eventName, decimal unitPrice,   string pictureUrl, int units = 1)
```

File: OrderApi/Models/OrderItem.cs [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+        {
+            if (units <= 0)
+            {
+                throw new OrderingDomainException("Invalid number of units");
+            }
+
+            ProductId = productId;
```
Comment (vlau26) [inline OrderApi/Models/OrderItem.cs]:
```suggestion
            EventId = eventId;
```

File: OrderApi/Models/OrderItem.cs [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+            ProductName = productName;
```
Comment (vlau26) [inline OrderApi/Models/OrderItem.cs]:
```suggestion
            EventName = eventName;
```

File: docker-compose.yml [hunk 6]
```diff
@@ -25,6 +25,27 @@ services:
...
 
+  order: 
+    build:
+      context: .\OrderApi
+      dockerfile: Dockerfile
+    image: eventbrite/orderapi
+    environment: 
+      IdentityUrl: http://10.10.10.30:5070
+      SQLSERVER_Host: catalogdb
+      SA_PASSWORD: ProductApi(!)
```
Comment (vlau26) [inline docker-compose.yml]: Could you actually just remove your changes to docker-compose so it doesn't overwrite what we currently have?

**Review by vlau26 — CHANGES_REQUESTED** [review]
a couple more minor changes.
Also, it doesn't seem like the migration files were generated for our project. Can you remove the preloaded migration files? We need to run migration ourselves for the auto-generated files to contain the appropriate properties for the tables.

File: docker-compose.yml (hunk 6 above)
Comment (rfeseha) [reply docker-compose.yml]: okay I will do that

Comment (rfeseha) [reply docker-compose.yml]: I didn't do migration thinking I would it after the PR but I will delete the ones that are already there 
```

**Pull request closed without merging**